packages/exhibitor/extra/zkServer.sh

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
```

Comment:
The only thing which _ever_ calls this script inside DC/OS is exhibitor. How are people doing weird things they might be depending upon in it? They aren't allowed to modify `/opt/mesosphere`. That means they aren't allowed to edit exhibitor, our systemd units, or any of the environment variables exhibitor is sending to this. They also aren't allowed to add jars to the folders this refers to.